app/clinic/index.html

```diff
@@ -35,7 +35,7 @@
       <!-- build:js scripts/vendor.clinic.min.js -->
       <!-- bower:js -->
       <script src="../../bower_components/jquery/dist/jquery.min.js"></script>
-      <script src="../../bower_components/angular/angular.min.js"></script>
```

Comment:
@ynurmahomed this is nice for development but not good in production, you can configure it in grunt to decide between min and non min.